.gitignore

```diff
@@ -34,6 +34,11 @@ fe/fe-core/gen
 fe/fe-common/.classpath
 fs_brokers/apache_hdfs_broker/src/main/resources/
 fs_brokers/apache_hdfs_broker/src/main/thrift/
+fe/fe-core/pseudo_cluster_*
```

Comment:
Should make the UT clear the files, not directly ignore it.

Reply:
fine.